src/Events/Integrations/Plugins/Colbri_Page_Builder/Provider.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\Colbri_Page_Builder;
+
+use TEC\Events\Integrations\Integration_Abstract;
+use TEC\Events\Integrations\Plugins\Plugin_Integration;
+
+/**
+ * Class Provider
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Colbri_Page_Builder
+ */
+class Provider extends Integration_Abstract {
+	use Plugin_Integration;
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_slug(): string {
+		return 'colibri-page-builder';
+	}
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @return bool Whether or not integrations should load.
+	 */
+	public function load_conditionals(): bool {
+		return class_exists( '\ColibriWP\PageBuilder\PageBuilder' );
```

Comment:
```suggestion
		return class_exists( '\ColibriWP\PageBuilder\PageBuilder', false );
```
We shouldn't autoload it unless we use it.